### PR TITLE
Fix triangle quadrature

### PR DESCRIPTION
**Pull request by akva2**
This fixes a typo in the 4 point triangle quadrature rule and adds a simple unit test.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: src/Utility/TriangleQuadrature.C [hunk 1]
```diff
@@ -42,7 +42,7 @@ static const double T4[8] = {
 };
 //! \brief 4-point rule weights.
 static const double W4[4] = {
- -0.5635,
+ -0.5625,
```
Comment (kmokstad) [inline src/Utility/TriangleQuadrature.C]: Most likely you're right, but double check with Tom Hughes' FEM book where these are taken from before approval.

Comment (akva2) [reply src/Utility/TriangleQuadrature.C]: yeah, i already did. also the sanity check should make it clear - the weights should sum to 1. they don't do currently, they do after the fix.

Comment (akva2) [reply src/Utility/TriangleQuadrature.C]: btw, in cause you wonder, the reason i ran into this is that coverage is now part of the static analysis jobs, and i noticed this class had no tests.

File: src/Utility/Test/TestTriangleQuadrature.C [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#include "TriangleQuadrature.h"
+
+#include "gtest/gtest.h"
+#include <numeric>
+
+
+TEST(TestTriangleQuadrature, Sanity)
+{
+  EXPECT_EQ(TriangleQuadrature::getCoord(-1), nullptr);
+  EXPECT_EQ(TriangleQuadrature::getWeight(-1), nullptr);
```
Comment (kmokstad) [inline src/Utility/Test/TestTriangleQuadrature.C]: Doesn't it make more sense to check getCoord(2)/getWeigth(2) instead, as there is no two-point rule?

File: src/Utility/Test/TestTriangleQuadrature.C [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  for (int i : {1,3,4}) {
+    const double* c = TriangleQuadrature::getCoord(i);
+    const double* w = TriangleQuadrature::getWeight(i);
+    double sum = std::accumulate(w, w+i, 0.0);
+    EXPECT_FLOAT_EQ(sum, 1.0);
+    for (int j = 0; j < i; ++j)
+      EXPECT_TRUE(c[j] >= 0 && c[j] <= 1.0);
```
Comment (kmokstad) [inline src/Utility/Test/TestTriangleQuadrature.C]: Cosmetic: 0.0 here, since 1.0

File: src/Utility/Test/TestTriangleQuadrature.C (hunk 2 above)
Comment (akva2) [reply src/Utility/Test/TestTriangleQuadrature.C]: i can do both.

File: src/Utility/Test/TestTriangleQuadrature.C (hunk 3 above)
Comment (akva2) [reply src/Utility/Test/TestTriangleQuadrature.C]: fixed

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**